samples/bot-message-reminder/csharp/README.md

```diff
@@ -0,0 +1,119 @@
+﻿﻿﻿
+## Message reminder with messaging extension action
+
+This sample shows a feature where user can schedule a task from messaging extension action and get a reminder card at a scheduled time.
+
+![Select message](MessagingExtensionReminder/Images/MessageAction.png)
```

Comment:
Let's keep only first three images at the top section which depicts the scenario for personal remider. Rest can go at the bottom of page. Let's do same for Node sampe as well.